test/mocha/release.js

```diff
@@ -22,10 +22,9 @@ describe("test/benchmark.js", function() {
         "-b",
         "-b bracketize",
         "-m",
-        "-mc passes=3",
-        "-mc passes=3,toplevel",
-        "-mc passes=3,unsafe",
-        "-mc keep_fargs=false,passes=3",
```

Comment:
These are `test/benchmark.js` options, whereas the time-out has been happening for `test/jetstream.js` ones instead.